src/bin/add_filter.rs

```diff
@@ -16,6 +16,11 @@ struct Cli {
     filter_file: String,
 }
 
+fn now_jd() -> f64 {
```

Comment:
You can use flare for this, it's just `flare::Time::now().to_jd()` or similar (plenty of examples of this in the code), so you don't have to deal with the jd conversion here again.